jsdoc-config.json

```diff
@@ -0,0 +1,33 @@
+{
+    "plugins": ["plugins/markdown",
+                "plugins/summarize"
+                // "./node_modules/@ckeditor/jsdoc-plugins/lib/export-fixer/export-fixer.js"
+                ],
+    "recurseDepth": 10,
+    "source": {
+        "include":[
+                    // The order here determines the order displayed in the docs.
```

Comment:
Can you remove all of these?

Reply:
Line 9 is a valid comment that conveys important information. I suggest leaving it in place.

Reply:
This is invalid for .json file, we are not allowed to have comments like this